api/protocol/README.md

```diff
@@ -0,0 +1,35 @@
+# 👉 RPC Protocol
+
+The **RPC protocol** used by Hemi's daemons is WebSocket-based and uses **JSON-encoded requests/responses**.
+
+The JSON data sent over WebSocket connections are referred to as **"messages"**.
+
+## 📩 Message Format
+
+An RPC message has the following format:
+
+```json
+{
+  "header": {
+    "command": "command-name",
+    "id": "request-id"
+  },
+  "payload": null
+}
+```
+
+### Header
+
+The message header contains metadata required for processing the message:
+
+- **`command`**: The name of the command being executed.
+- **`id`**: A string used to uniquely identify each request. Responses will have the same `id` as the request, making it
+  possible to match requests to responses.
+
+> [!NOTE]
```

Comment:
Do we actually need this point here?  As long as our rpc doesn't care, it really is up to the user.  I frequently use non-random json ids to identify what a respond is and not just to match up with the initial request (when stress testing for example).  Seems like it should be up to the user and we shouldn't say anything here.

Reply:
I think I mentioned this as more of a recommendation than a requirement.
I think most users will be able to figure out that it should be unique when creating an API client, however, so I think removing this would be fine. Do you think we should remove it?

Reply:
I'd remove in the spirit of not having the keeping the docs a little leaner.

Reply:
Okay, removed